app/models/adventure.rb

```diff
@@ -1,13 +1,15 @@
 class Adventure < ActiveRecord::Base
-  belongs_to :bucket_list
+  has_many :bucket_list_adventures#, dependent: :destroy, inverse_of: :bucket_lists
```

Comment:
Line is too long. [83/80]<br>Missing space after #.<br>Put a space before an end-of-line comment.